tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -3757,6 +3771,42 @@ int BPF_KPROBE(trace_mprotect_alert)
     return 0;
 }
 
+SEC("raw_tracepoint/sys_init_module")
+int syscall__init_module(void *ctx)
+{
+    event_data_t data = {};
+    if (!init_event_data(&data, ctx))
+        return 0;
+
+    syscall_data_t *sys = bpf_map_lookup_elem(&syscall_data_map, &data.context.host_tid);
+    if (!sys)
+        return -1;
+
+    bin_args_t bin_args = {};
+
+    u32 pid = data.context.host_pid;
+    u32 dev = 0;
+    u64 inode = 0;
```

Comment:
No meaning for having two variables here, as these values don't change (constant zero).
You can have `u64 dummy = 0` and use it for both bpf_probe_read below